client/test/Completions3_0.test.ts

```diff
@@ -26,9 +23,6 @@ describe('Completions 3.0', () => {
     });
 
     beforeEach(async () => {
-        const filePath = path.join(basicRazorApp30Root, 'Pages', 'Index.cshtml');
```

Comment:
Had to move these down to each test because when you do `showTextDocument` it unshows any previous documents and kills their editors. So since we have 2 editors we care about when we would show the .razor file it would close out the .cshtml doc.